src/patterns/strategy/discount-example/example/example.exec.ts

```diff
@@ -0,0 +1,40 @@
+import {clientApp} from "./example.impl";
+
+const RANDOM_PRICE = 1000;
+
+// Calculating a final price using a fixed amount discount
+clientApp(RANDOM_PRICE, {
+  price: RANDOM_PRICE,
```

Comment:
Everything looks good
Looks like the price is duplicating, as the first argument and insider order
It seems to me that having price in order should be enough